src/UploadButtonRenderer.php

```diff
@@ -89,6 +89,9 @@ protected function getHtml( ParameterProvider $paramProvider ) {
 
 		return
 
+			'<label for="wfUploadDescription">' . \Message::newFromKey( 'upload-form-label-infoform-description' )->escaped() . ':</label><br>'.
+			'<span class="mw-input"><textarea name="wfUploadDescription" id="wfUploadDescription" cols="80" rows="8">' .
+			$escapedUploadPageText . '</textarea><br> ' .
```

Comment:
I'd be more confident in this code if the escaping was done via `Html::element`. The context in which the value is placed changed in this PR. It was a HTML attribute and now it is a value in a tag. I'm not sure the return value of `getEscapedUploadPageText` is safe for the latter.